### DISCUSSION TITLE
Birth cohorts?

### DISCUSSION THREAD

**Discussion by ECHILD-demo** [UCL-ECHILD/ECHILD#6] [ECHILD Q&A] [answered]
I'm interesting in creating birth cohorts using ECHILD. Does anyone have any ideas about how to do this?

**Comment by echild-admin** [comment, marked as answer]
Take a look at the [ECHILD Phenotype Code List Repository](https://code.echild.ac.uk) (developed by @matthewjay15). 

@AniaZylb has published on methods for developing birth cohorts using HES data available in ECHILD, see https://code.echild.ac.uk/births_zylb_v1
